enterprise/utils.py

```diff
@@ -1431,6 +1435,11 @@ def enroll_users_in_course(
                     course_id,
                     role=MANUAL_ENROLLMENT_ROLE,
                 )
+
+            if license_uuids:
+                # create licensed enrollment if a subscription license is supplied
+                enterprise_customer_user = get_enterprise_customer_user(user.id, enterprise_customer.uuid)
+                get_create_ent_enrollment(course_id, enterprise_customer_user, license_uuids.get(user).get(course_id))
```

Comment:
if we have license uuids, this is a bulk subscription enrollment request and all users are assumed to have licenses for all requested courses.

Reply:
This comment doesn't belong in the PR: it's part of the design spec you've agreed on. Add it to the doc string parameters. Also, if you are saying "if any license_uuids are included, all of them in the bulk enrollment MUST have one, then you need a better error message to return: you are going to have to validate them all beforehand and if one user is missing a license, then stop the entire transaction.

Reply:
Also it looks like this isn't actually required based on the lines of code below it?